src/api/streams/updateStream.js

```diff
@@ -2,6 +2,13 @@
 import type { ApiResponse, Auth } from '../transportTypes';
 import { apiPatch } from '../apiFetch';
 
+/**
+ * https://zulip.com/api/update-stream
+ *
+ * NB the caller must adapt for old-server compatibility; see comment.
+ */
+// TODO(#4659): Once we pass the feature level to API methods, this one
+//   should encapsulate a switch at feature level 64.  See its call sites.
```

Comment:
Could mention that we have a plan sketched out for that at https://github.com/zulip/zulip-mobile/issues/4659#issuecomment-914996061.

Reply:
I think I'm content to let the issue number stand for that. If in the future when we go to implement that issue we overlook that feature, then when we look at this TODO comment and go looking for how the issue was meant to relate to feature levels I think we'll find those comments.

Reply:
Oh, right, makes sense! I read right past the issue number in the comment, without noticing it. Then I went and dug up the link to the specific comment, just because I remembered writing it.